packages/fuel-gauge/src/contract.test.ts

```diff
@@ -132,21 +132,10 @@ describe('Contract', () => {
       cache: false,
     });
 
-    const scope = contract.functions
-      .call_external_foo(1336, otherContract.id)
-      .addContracts([otherContract.id]);
-
-    expect(scope.transactionRequest.getContractInputs()).toEqual([
-      { contractId: contract.id.toB256(), type: 1, txPointer },
-      { contractId: otherContract.id.toB256(), type: 1, txPointer },
-    ]);
-
-    expect(scope.transactionRequest.getContractOutputs()).toEqual([
-      { type: 1, inputIndex: 0 },
-      { type: 1, inputIndex: 1 },
-    ]);
+    const scope = contract.functions.call_external_foo(1336, otherContract.id);
```

Comment:
I think this is sufficient for tests following the pattern [here](https://github.com/FuelLabs/fuels-rs/pull/628/files#diff-99029e6dd80e10a946c69a9ea3b16c35afb83813b457d3bdbddc4905e4c1aeea).

The only issue is that the underlying transaction request doesn't get mutated if there are updates on to it so the `getContract`  assertion is no longer needed.  I'm going to create follow-up issue to see if we should update the `scope.transactionRequest`